recipes/nearpy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "NearPy" %}
+{% set version = "1.0.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "???" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  # No sdist available
+  # xref: https://github.com/pixelogik/NearPy/issues/67
+  # fn: '{{ name }}-{{ version }}.tar.gz'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # '{{ hash_type }}': '{{ hash_value }}'
+  # ref: https://github.com/pixelogik/NearPy/issues/67
+  git_url: https://github.com/pixelogik/NearPy
+  git_revision: 1.0.0
```

Comment:
Use  the tarball from github. https://github.com/pixelogik/NearPy/releases

Reply:
is that stable now? doesn't github change hashes on us every now and then?